### PR TITLE
Add mirage-profile{,-unix,-xen}.0.8.0

### PR DESCRIPTION
**Pull request by djs55**
Signed-off-by: David Scott <dave@recoil.org>

### CODE REVIEW

**Comment by talex5** [comment]
Looks like the names are wrong. I've made a PR to fix it upstream: https://github.com/mirage/mirage-profile/pull/27/commits

File: packages/mirage-profile-unix/mirage-profile-unix.0.8.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "mirage-profile"
```
Comment (avsm) [inline packages/mirage-profile-unix/mirage-profile-unix.0.8.0/opam]: wrong name...

**Comment by djs55** [comment]
I think this is ok now
- `name` removed from metadata
- `tcpip.3.1.3` fails to build with a `cstruct.ppx` problem (hopefully fixed in latest master)
- `mirage-xen-minios.0.8.0` fails to build on Alpine because of a missing C header
- a download failed on Ubuntu, presumed temporary

**Comment by avsm** [comment]
ive pushed a merge to see if the new cstruct fixes the tcpip revdep failure

**Comment by avsm** [comment]
actually not - can you merge @djs55 

```
remote: Resolving deltas: 100% (37/37), completed with 8 local objects.
To github.com:djs55/opam-repository
 ! [remote rejected] mirage-profile.0.8.0 -> mirage-profile.0.8.0 (permission denied)
error: failed to push some refs to 'git@github.com:djs55/opam-repository'
```

**Comment by avsm** [comment]
#9480 is merged which holds back tcpip, so i think this is also safe to merge

**Pull request merged**